doc/guide/pd/polydispersity.rst

```diff
@@ -533,6 +609,21 @@ T Allen, in *Particle Size Measurement*, 4th Edition, Chapman & Hall, London (19
 
 .. ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
 
+Related sections
+^^^^^^^^^^^^^^^^
+
+See also:
+
+:ref:`PStheory`
+
+:ref:`Resolution_Smearing`
+
+:ref:`Interaction_Models`
+
+:ref:`orientation`
```

Comment:
Capital Letter?

Reply:
`orientation` is just a label and is not seen when the docs are rendered; what is rendered is the section heading the label refers to: 'Oriented Particles'.
![image](https://user-images.githubusercontent.com/10938679/198987843-91378dfe-5eae-4269-a640-8793dbcae15d.png)
To change the capitalisation of this label will mean editing ~dozen model files in addition to a handful of doc pages, so I'm respectfully going to say no to this. :-)